importexport/musicxml/importmxmlpass2.h

```diff
@@ -183,7 +193,8 @@ class MusicXMLParserNotations {
       int tremoloNr() const { return _tremoloNr; }
       bool mustStopGraceAFter() const { return _slurStop || _wavyLineStop; }
 private:
-      void addTechnical(Note* note);
+      void harmonic();
+      void addTechnical(const GenericAnnotation& notation, Note* note);
```

Comment:
I don’t quite understand by name what this method does

maybe there is a more suitable name? What do you think?

Reply:
i do agree that by only looking into the name it does not help much, but i only followed the names used there. i believe that you can understand within the context that this is the XML hierarchy `note/notations/technical/harmonic` and the method of class `MusicXMLParserNotations` so we are worried on parsing the children of Notations group. I could be more explicit and rename every method like this: `parseHarmonicElement`, `parseTechnicalGroup`, `parseArticulationsElement`, etc